src/amo/components/CollectionManager/index.js

```diff
@@ -77,13 +78,16 @@ type State = {|
 |};
 
 export class CollectionManagerBase extends React.Component<Props, State> {
+  timeout: TimeoutID;
+
   static defaultProps = {
     setTimeout: typeof window !== 'undefined' ? window.setTimeout.bind(window) : () => {},
   };
 
   constructor(props: Props) {
     super(props);
     this.state = this.propsToState(props);
+    this.timeout; // eslint-disable-line no-unused-expressions
```

Comment:
Do you really need this?

Reply:
I thought we did but we don't it seems : )